include/aspect/material_model/interface.h

```diff
@@ -274,6 +274,18 @@ namespace aspect
       std::vector<double> viscosities;
 
       /**
+       * Constitutive tensors at the given positions. This variable
+       * can be used to implement exotic rheologies such as anisotropic
+       * viscosity.
+       *
+       * @note The strain rate term in equation (1) of the manual will be
+       * multiplied by this tensor *and* the viscosity scalar (\\eta). If no
+       * value is assigned here the identity tensor will be used in its place.
+       * This leaves the isotropic constitutive law unchanged by default.
+       */
+      std::vector<SymmetricTensor<4,dim> > constitutive_tensors;
```

Comment:
By the way, what will this variable be set to "if no value is assigned here"? Does that mean the vector is simply empty, or that it has the right size and simply has special values?